modules/platforms/cpp/ignite/client/detail/table/tables_impl.h

```diff
@@ -55,6 +53,16 @@ class tables_impl {
      */
     void get_table_async(std::string_view name, ignite_callback<std::optional<table>> callback);
 
+    /**
+     * Gets a table by name.
+     * See Table::get_table_async() for details.
```

Comment:
Is there really a `Table::get_table_async()` method? Looks suspicious.

Reply:
You are right. Fixed.